src/lib/in.c

```diff
@@ -567,19 +567,6 @@ pixelmouse_click(inputctx* ictx, ncinput* ni, long y, long x){
   x /= ictx->ti->cellpxx;
   x -= ictx->lmargin;
   y -= ictx->tmargin;
-  // convert from 1- to 0-indexing, and account for margins
```

Comment:
ugh, i should really probably have factored these out. if you're getting rid of them, i guess it doesn't matter.